tests/e2e/support/objects/app-files/resource/actions.ts

```diff
@@ -1007,6 +1007,7 @@ export const getDisplayedResourcesFromSearch = async (page): Promise<string[]> =
 export const getDisplayedResourcesFromFilesList = async (page): Promise<string[]> => {
   const files = []
   await page.locator('[data-test-resource-path]').first().waitFor()
+  await new Promise((resolve) => setTimeout(resolve, 1000))
```

Comment:
Maybe waitForResponse after search in following function will work https://github.com/owncloud/web/blob/992f52250492f6645db6daa2eb7564ff4af4b6c1/tests/e2e/support/objects/app-files/search/actions.ts#L21-L25

Reply:
nice idea, I'll try that

Reply:
sadly didn't work but I'll keep this check